build.gradle

```diff
@@ -14,9 +17,61 @@ repositories {
 
 dependencies {
     api("org.openmicroscopy:omero-blitz:5.5.0-m7")
+    api("ome:formats-api:5.9.2") {
+        exclude group: "org.perf4j", module: "perf4j"
+    }
     implementation("commons-beanutils:commons-beanutils:1.9.3")
     implementation("com.zeroc:icegrid:3.6.4")
     testCompile("org.testng:testng:6.9.8")
+    // Re-activate from transitive excludes
+    //implementation("org.apache.commons:commons-collections4:4.1")
+    implementation("commons-collections:commons-collections:3.2.2")
+    implementation("commons-lang:commons-lang:2.6")
+    implementation("com.google.guava:guava:17.0")
```

Comment:
why are we reading guava 17?

Reply:
This is coming from a combination of Chris' previous work as well as the state of the build as of 2-3 weeks ago. If you know it's not needed, feel free to update.